### ISSUE TITLE
Support a Soundcard Mixer for multichannel I/O interfaces on all platforms

### ISSUE THREAD

**Issue by matthimue** [jamulussoftware/jamulus#177]
There have been number of discussions and requests of how to integrate a 'Soundcard Mixer' into jamulus. Discussions I had have been listed in https://github.com/corrados/jamulus/issues/151 (going to be closed and replaced now by this) and I'd like to summmarize the ideas here in cheatsheet for the UI.

Main idea is to have an interface where you can configure your N input channels and N outputchannels like you are used on an analog mixing console. Inputs are 'panned' to Jamulus server inputs and outputs are selected from jamulus server outputs with switches that allow configuration of MONO/STEREO L/R for your amps connected to the jamulus server.
Another import. So you are able to configure all I/Os of your individual soundcard.

Another thing to realize is, that ASIO on windows is function comparable to JACK on Linux, it's just an audio host. So this is the only platform dependent. This having in mind, you can have a UI approach, that is usable on all platforms.

To summarize what I mean, I draw a 'photoshopped' tweak cheatsheet of the UI, that can provide functionality for a number of (feature) requests to jamulus which saw in Issue list.

The cheatsheet should be self explaining.
![Jamulus-Mixer](https://user-images.githubusercontent.com/64200197/81143678-aecdfb80-8f72-11ea-96c3-66f794b524b4.jpg)

**Comment by corrados** [comment]
Here is the link to the Git branch: https://github.com/corrados/jamulus/tree/feature_sndcrdmixer

**Comment by ann0see** [comment]
Hi, Sorry for the maintenance noise here. I’m just into triaging issues.
This issue is a feature request nobody is working on right now (or anymore). Usually these kind of issues either need some more discussion (open a discussion on GitHub discussions) or haven’t been clarified enough. If there’s still a strong need to implement this feature, please open a new issue.
Therefore and to keep the issues list short, I will close this issue now. Please re-open it if necessary.

**Comment by ann0see** [comment]
Actually, maybe @pgScorpio or @dcorson-ticino-com worked on it, however, I think it would be worth opening another issue to continue.

**Comment by pgScorpio** [comment]
@ann0see 
Yes in my sound redesign mixing of ch1 and/or ch2 with any one of the channels 3 or higher will be supported for any interface with more than 2 channels.

**Comment by ann0see** [comment]
Thanks for your clarification. ;-)
